api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -192,6 +192,15 @@ public ResponseEntity<Workspace> createWorkspace(Workspace workspace) throws Bad
     }
     final Workspace createdWorkspace = workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace);
     workspaceAuditor.fireCreateAction(createdWorkspace, dbWorkspace.getWorkspaceId());
+
+    if (cdrVersion.getTanagraEnabled()) {
+      try {
+        workspaceService.createTanagraStudy(
+            createdWorkspace.getNamespace(), createdWorkspace.getName());
+      } catch (Exception e) {
+        log.log(Level.SEVERE, "Could not create a Tanagra Study.", e);
+      }
+    }
     return ResponseEntity.ok(createdWorkspace);
```

Comment:
If creation of study fails -> should we continue? Because we already created the workspace that is in a bad state. so in the catch(Exception e) {} block, if study fails to create should we not `DELETE` the newly created workspace? and throw an error -> "was able to create Aou-workspace but study could not be created try later. Deleting newly created workspoace?"